client/src/modules/assets/templates/action.cell.html

```diff
@@ -47,6 +47,12 @@
       </a>
     </li>
 
+    <li>
+      <a data-method="view-scan-history" ng-click="grid.appScope.showAssetScans(row.entity)" href>
```

Comment:
Note that you can also do this with a `ui-sref` and save the need to have an `ng-click` with a scoped binding.  I don't know which is better in terms of performance though.

Reply:
Thanks.  I'll try it next time.